js/pihole/index.js

```diff
@@ -50,32 +32,74 @@ $(document).ready(function() {
     };
     var animate = false;
     var ctx = document.getElementById("queryOverTimeChart").getContext("2d");
-    timeLineChart = new Chart(ctx).Line(chartData,
-        {
-            pointDot : false,
-            legendTemplate : "<ul class=\"<%=name.toLowerCase()%>-legend\"><% for (var i=0; i<datasets.length; i++){%><li><span style=\"background-color:<%=datasets[i].strokeColor%>\"></span><%if(datasets[i].label){%><%=datasets[i].label%><%}%></li><%}%></ul>",
-            animation : animate
+    timeLineChart = new Chart(ctx, {
```

Comment:
The properties of the "timeLineChart" have now to be set in a very different format.